### PR TITLE
add a Dictionary Passing Project Group

### PR DESCRIPTION
**Pull request by lcnr**
Mainly to have a separate zulip stream and to more easily collect the open problems and interesting tests/edge-cases in github issues

cc @rust-lang/types @Nadrieril @nikomatsakis @BennoLossin

### CODE REVIEW

File: teams/project-dictionary-passing.toml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    "nikomatsakis",
+    "BennoLossin"
+]
+alumni = []
+
+[website]
+name = "Meta-language Dictionary Passing Working Group"
+description = "Modelling the trait system and cycle handling via meta-language dictionary passing"
+repo = "https://github.com/rust-lang/project-dictionary-passing"
+zulip-stream = "T-types/dictionary-passing"
```
Comment (jieyouxu) [inline teams/project-dictionary-passing.toml]: Nit: do you want this to be lowercase? I.e. the T-types stream is called [#t-types](https://rust-lang.zulipchat.com/#narrow/channel/144729-t-types)

```toml
zulip-stream = "t-types/dictionary-passing"
```

Comment (Nadrieril) [reply teams/project-dictionary-passing.toml]: yes plz

**Comment by BoxyUwU** [comment]
hell yeah!

File: teams/project-dictionary-passing.toml (hunk 1 above)
Comment (lcnr) [reply teams/project-dictionary-passing.toml]: looks like we automatically convert to lowercase. Assumptions on binders used uppercase but on zulip it already is lowercase?

Comment (jieyouxu) [reply teams/project-dictionary-passing.toml]: Huh, maybe

**Review by jieyouxu — APPROVED** [review]
